posts/helpers.py

```diff
@@ -164,8 +169,22 @@ def get_external_post(post_id, requestor):
             author = UserSerializer(data=post['author'])
             author = author.to_user_model()
             post_model = Post(title=post['title'], content=post['content'], contentType=post['contentType'],
-                              description=post['description'], id=post['id'], author=author)
-            return post_model
+                              description=post['description'], id=post['id'], author=author, source=post['source'],
+                              origin=post['origin'])
+            comment_list = []
+            for comment in post['comments']:
+                # CHeck if paginated
+                commenter_wwuser = WWUser.objects.get_or_create(url=comment['author']['url'],
```

Comment:
we don't need to check if the user is local because we're working on the assumption all local users have a ww_user with local already defined and external users aren't local

Reply:
sorry this is right I'm just making stating the assumption you're making